lib/models/permiso_covid.dart

```diff
@@ -0,0 +1,39 @@
+class PermisoCovid {
+  String? id;
+  String? perfil;
+  String? motivo;
+  String? campus;
+  String? dependencia;
+  String? jornada;
+  String? fechaSolicitud;
+
+  PermisoCovid(
+    this.id,
+    this.perfil,
+    this.motivo,
+    this.campus,
+    this.dependencia,
+    this.jornada,
+    this.fechaSolicitud,
+  );
+
+  factory PermisoCovid.fromJson(Map<String, dynamic> json) {
+    return PermisoCovid(
+      json['id'],
+      json['perfil'],
+      json['motivo'],
+      json['campus'],
+      json['dependencia'],
+      json['jornada'],
+      json['fechaSolicitud'],
+    );
+  }
+
+  static List<PermisoCovid> fromJsonList(List<Map<String, dynamic>> json) {
+    List<PermisoCovid> lista = [];
```

Comment:
Quizás falta la comprobación de si la lista es `null` como en los otros modelos
```suggestion
    if (json == null) {
      return [];
    }
    List<PermisoCovid> lista = [];
```

Reply:
No estoy seguro si es necesario. La única forma que se me ocurre que llegue a ser null es que falle la petición, pero se supone que 'algo' atrapa esa condición antes siempre, no?

Reply:
Tru, lo marco como resuelto